extensions/wikia/AffiliateService/js/tracker.js

```diff
@@ -0,0 +1,57 @@
+define('ext.wikia.AffiliateService.tracker', [
+	'jquery',
+	'wikia.tracker',
+	'wikia.window',
+], function ($, tracker, win) {
+	'use strict';
+
+	var defaultOptions = {
+		campaignId: '',
+		categoryId: '',
+		extraTracking: {},
+	}
+
+	function track(action, label, options) {
+		var currentOptions = $.extend(defaultOptions, options);
+
+		// convert extra tracking options
+		var extraTracking = [];
+		options.extraTracking.forEach((kv) => {
+			extraTracking[`affiliation_${kv.key}`] = kv.val;
```

Comment:
Template literals aren't yet supported in Edge AFAIK.

Reply:
They might be supported in Edge, but they surely aren't supported in IE11, which is also a [supported browser](https://c.fandom.com/wiki/Help:Supported_browsers). Since this is probably going to be bundled with most other JavaScript via ResourceLoader, I'm guessing it would make a lot more scripts error out like it [historically](https://github.com/Wikia/app/pull/12709) happened [when](https://github.com/Wikia/app/pull/15165) ES6 [features](https://github.com/Wikia/app/pull/15235) were added to the code.

Reply:
Sorry, this shouldn't be template literal in the first place

Reply:
Fixed